spec/models/manageiq/providers/vmware/network_manager/refresher_spec.rb

```diff
@@ -0,0 +1,127 @@
+describe ManageIQ::Providers::Vmware::NetworkManager::Refresher do
+  before do
+    @host = Rails.application.secrets.vmware_cloud.try(:[], 'host') || 'vmwarecloudhost'
+    host_uri = URI.parse("https://#{@host}")
+
+    @hostname = host_uri.host
+    @port = host_uri.port == 443 ? nil : host_uri.port
+
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems = FactoryGirl.create(
+      :ems_vmware_cloud,
+      :zone     => zone,
+      :hostname => @hostname,
+      :port     => @port
+    )
+    @ems_network = @ems.network_manager
+    @network_type_vdc = "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVDCNet"
+    @network_type_vapp = "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::VappNet"
+
+    @userid = Rails.application.secrets.vmware_cloud.try(:[], 'userid') || 'VMWARE_CLOUD_USERID'
+    @password = Rails.application.secrets.vmware_cloud.try(:[], 'password') || 'VMWARE_CLOUD_PASSWORD'
+
+    VCR.configure do |c|
+      # workaround for escaping host in spec/spec_helper.rb
+      c.before_playback do |interaction|
+        interaction.filter!(CGI.escape(@host), @host)
+        interaction.filter!(CGI.escape('VMWARE_CLOUD_HOST'), 'vmwarecloudhost')
+      end
+
+      c.filter_sensitive_data('VMWARE_CLOUD_AUTHORIZATION') { Base64.encode64("#{@userid}:#{@password}").chomp }
+    end
+
+    cred = {
+      :userid   => @userid,
+      :password => @password
+    }
+
+    @ems.authentications << FactoryGirl.create(:authentication, cred)
+  end
+
+  it ".ems_type" do
+    expect(described_class.ems_type).to eq(:vmware_cloud_network)
+  end
+
+  it "will perform a full refresh" do
+    2.times do # Run twice to verify that a second run with existing data does not change anything
+      @ems.reload
+      @ems_network.reload
+      VCR.use_cassette(described_class.name.underscore, :allow_unused_http_interactions => true) do
+        EmsRefresh.refresh(@ems)
+        EmsRefresh.refresh(@ems_network)
```

Comment:
I just tried this on GCE and see the same thing, so it is normal (not sure if expected :smile:)